locale/coa/ar/Personal.xml

```diff
@@ -0,0 +1,269 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration xmlns="http://ledgersmb.org/xml-schemas/configuration">
+  <coa>
```

Comment:
Could the language be in the XML instead of only in the path? At least descriptions can be spell checked.

Reply:
Actually, the indication in the path is supposed to be a region or jurisdiction, not a language name. That manifests in e.g. the chart for Canada which has the same chart, but in 2 languages.
But yes, we can add the language to the xml. I'll need to create an attribute for that then. Do we know the language of each of the provided CoAs?